3_core_core_analysis/1_core_core_relationships_across_strains.py

```diff
@@ -275,66 +256,74 @@ def compare_gene_relationships(gene_mapping_dict, mapping_to, pao1_corr, pa14_co
     "Transcriptional similarity across strains", ascending=False
 ).tail(round(least_percent * len(pao1_corr_df)))
 
+pao1_most_threshold = pao1_most_stable.iloc[-1][
+    "Transcriptional similarity across strains"
+]
+pao1_least_threshold = pao1_least_stable.iloc[0][
+    "Transcriptional similarity across strains"
+]
+print(pao1_least_threshold, pao1_most_threshold)
+
 pa14_most_stable = pa14_corr_df.sort_values(
     "Transcriptional similarity across strains", ascending=False
 ).head(round(most_percent * len(pa14_corr_df)))
 pa14_least_stable = pa14_corr_df.sort_values(
     "Transcriptional similarity across strains", ascending=False
 ).tail(round(least_percent * len(pa14_corr_df)))
 
+pa14_most_threshold = pa14_most_stable.iloc[-1][
```

Comment:
Yes, by definition core genes are those genes that are homologous across strains. However based on the venn diagrams there are a couple of core genes that are found in one strain type and not the other just because that core gene wasn't measured in the other strain type or our similarity scores were just slightly different mapping from pao1>pa14 versus pa14>pao1. But there are only 2 cases of this so its not worrisome.